guides/common/assembly_discovering-hosts-on-a-network.adoc

```diff
@@ -0,0 +1,50 @@
+include::modules/con_discovering-hosts-on-a-network.adoc[]
+
+include::modules/con_prerequisites-for-using-discovery.adoc[leveloffset=+1]
+
+include::modules/proc_installing-the-discovery-service.adoc[leveloffset=+1]
+
+// PXE Discovery
+include::modules/con_discovery-in-pxe-mode.adoc[leveloffset=+1]
+
+include::modules/con_discovery-workflow-in-pxe-mode.adoc[leveloffset=+2]
+
+include::modules/proc_setting-discovery-as-the-default-PXE-boot-option.adoc[leveloffset=+2]
+
+include::modules/proc_performing-discovery-in-pxe-mode.adoc[leveloffset=+2]
+
+include::modules/proc_customizing-the-discovery-pxe-boot.adoc[leveloffset=+2]
+
+ifdef::satellite[]
+include::modules/con_discovering-hosts-from-multiple-smartproxyservers.adoc[leveloffset=+2]
+endif::[]
+
+// PXE-less Discovery
+include::modules/con_discovery-in-pxeless-mode.adoc[leveloffset=+1]
+
+include::modules/con_discovery-workflow-in-pxeless-mode.adoc[leveloffset=+2]
+
+include::modules/proc_performing-discovery-in-pxeless-mode.adoc[leveloffset=+2]
+
+include::modules/proc_customizing-the-discovery-iso.adoc[leveloffset=+2]
+
+// Common next steps
+include::modules/ref_automatic-contexts-for-discovered-hosts.adoc[leveloffset=+1]
+
+include::modules/proc_creating-hosts-from-discovered-hosts.adoc[leveloffset=+1]
+
+include::modules/proc_creating-discovery-rules.adoc[leveloffset=+1]
+
+include::modules/proc_extending-the-discovery-image.adoc[leveloffset=+1]
+
+ifndef::satellite[]
+include::modules/proc_building-a-discovery-image.adoc[leveloffset=+1]
+
+include::modules/proc_preparing-discovery-image-for-pxe-boot.adoc[leveloffset=+1]
+
+include::modules/proc_making-the-custom-discovery-iso-bootable.adoc[leveloffset=+1]
+endif::[]
+
+include::modules/ref_kernel-parameters-for-discovery-customization.adoc[leveloffset=+1]
+
+include::modules/ref_troubleshooting-discovery.adoc[leveloffset=+1]
```

Comment:
I know that you did not update this module. It could use reorganization if you have time.

Reply:
I agree. I'm planning it with another round of improvements in another PR.

Reply:
Created an issue: https://github.com/theforeman/foreman-documentation/issues/3041